diff --git a/CMakeLists.txt b/CMakeLists.txt
index 96a2692e..05b836f1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,7 +49,7 @@ endif()
 include("dependencies.cmake")
 
 # Include the common build rules for C shared utility
-include(${SHARED_UTIL_FOLDER}/configs/azure_iot_build_rules.cmake)
+include(deps/azure-c-shared-utility/configs/azure_iot_build_rules.cmake)
 
 set_platform_files(${SHARED_UTIL_FOLDER})
 